ci/run_easymesh_cert.sh

```diff
@@ -121,16 +121,16 @@ main() {
 
     if [ -n "$OWNCLOUD_UPLOAD" ]; then
         if is_prplmesh_device "$TARGET_DEVICE"; then
-            REMOTE_PATH="$TARGET_DEVICE/$(basename "$LOG_FOLDER")"
+            REMOTE_PATH="$TARGET_DEVICE"
         else
-            REMOTE_PATH="certified/$TARGET_DEVICE/$(basename "$LOG_FOLDER")"
+            REMOTE_PATH="certified/$TARGET_DEVICE"
         fi
         info "Uploading $LOG_FOLDER to $OWNCLOUD_PATH/$REMOTE_PATH"
-        "$scriptdir"/owncloud/upload_to_owncloud.sh "$OWNCLOUD_PATH/$REMOTE_PATH" "$LOG_FOLDER" || {
```

Comment:
Ok here you explain why you want the "direct" option.
Note that even if you do partial uploads to a temporary directory, you can still move the incomplete folder at the end if you want.

IMO, if the default behavior doesn't correspond to what you expect (i.e. it uploads by replacing the remote path you specified, instead of uploading /inside/ the remote path) we should change it instead of adding the `direct` option as a workaround.

To me you can merge it as it is now though.

Reply:
I didn't want to change it so not to cause issues with the CI uploads. I think doing it in steps, and later removing the direct option is better.